src/pkg/backup/details/details.go

```diff
@@ -367,21 +367,22 @@ type OneDriveInfo struct {
 	ParentPath string    `json:"parentPath"`
 	ItemName   string    `json:"itemName"`
 	Size       int64     `json:"size,omitempty"`
+	Owner      string    `json:"owner,omitempty"`
 	Created    time.Time `json:"created,omitempty"`
 	Modified   time.Time `json:"modified,omitempty"`
 }
 
 // Headers returns the human-readable names of properties in a OneDriveInfo
 // for printing out to a terminal in a columnar display.
 func (i OneDriveInfo) Headers() []string {
-	return []string{"ItemName", "ParentPath", "Size", "Created", "Modified"}
+	return []string{"ItemName", "ParentPath", "Size", "Owner", "Created", "Modified"}
```

Comment:
Does this change need to be reflected in any docs?

Reply:
I think we're good - I don't see any reference to details anywhere.

Reply:
The doc examples are for exchange so this is ok